docs/cep/query-guide/query.md

```diff
@@ -955,15 +955,15 @@ select e1.roomNo as roomNo
 from e1=RegulatorStateChangeStream[action == 'start'] -> not TempStream[e1.roomNo == roomNo and temp < 12] and e2=RegulatorStateChangeStream[action == 'off'];
 ```
 
-This Stream Application generates an alert if the temperature does not reduce to 12 degrees within 5 minutes of switching on the regulator.  
+This Stream Worker generates an alert if the temperature does not reduce to 12 degrees within 5 minutes of switching on the regulator.  
```

Comment:
Write out numbers zero through ten as words.

Reply:
@diana-macrometa could you explain to me where would you like me to do that since I'm not sure I understand the task?

Reply:
Replace "5" with "five."